src/auth/dto/password-reset.dto.ts

```diff
@@ -0,0 +1,13 @@
+import{ IsEmail, IsNotEmpty } from 'class-validator';
+
+export class PasswordResetDto {
+    @IsEmail()
+    @IsNotEmpty()
+    email: string;
+
```

Comment:
We don't need the email in the password reset. We are assuming the user is redirected from their email address.

```suggestion
```